scratch-render/rspack.config.js

```
@@ -0,0 +1,105 @@
+const path = require("path");
+const { rspack } = require("@rspack/core");
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)